src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -88,6 +88,19 @@ public ApiMethodView getApiMethodView(Interface service, Method method) {
     return grpcStubTransformer.generateGrpcStubs(context);
   }
 
+  private String getStubNameFor(Interface service, Method method) {
+    String jsMethodName = upperCamelToLowerCamel(method.getSimpleName());
```

Comment:
Optional: this seems strange -- does this duplicate existing logic to transform a `Method` instance into its NodeJS name? (Since clearly this transformation has already happened elsewhere in order for the method name to be attached to the `GrpcStubView`...)